mdp.ipynb

```diff
@@ -435,14 +423,42 @@
    },
    "outputs": [
     {
+     "name": "stderr",
```

Comment:
Can these errors be avoided?
I get the same error but only once. Maybe removing it through text editor could work as a temporary fix